### DISCUSSION TITLE
"Error Disconnected" Message whenever I try to save certain settings..

### DISCUSSION THREAD

**Discussion by ohd22** [meshtastic/Discussions#151] [Android]
Over the past month or so, every time I attempt to update and save my location coordinates, I get a "Error Disaconnected" message and the updates don't take effect (none of my devices have GPS, so all coordinates are entered manually).  This happens with my Heltec V3's as well as my RAK Wisblock.  Othe configurstions such as "Channels", "Device" & "Network" (for example) update just fine.  I'm assuming that the common denominator here is the App, but I have yet to figure this out.  Any suggestions would be appreciated

**Comment by garthvh** [comment]
Some settings require a reboot and the connected app will disconnect when that happens.

**Reply by LCuinu** [reply]
Update.... I was not aware of the position precision implementation, apologies.  Now switched on for our private channel.  When I got the error disconnected message and saw my ocation off by 2.9km I thought it hadn't taken.

Same problem on two RAK 4631.  Entered a manual coordinate via bluetooth (using as routers in the hills).  Initially the oordinates seem to take but after a while they change to a random location but within about 2.9km!  When I ask for 'request position' (when linked to another device) the position is correct.  But after a time it changes back to a location I've never ever visited or had aquaintance with (but quite near...about 2.9km).  The first RAK I thought 'oh I'll just delete the DB on both' but it comes back!?  All devices are on beta 2.5.15.7.  Strangest thing.  The second brand new unit setup tonight is in the middle of a Loch...until I request position but then goes back to the Loch after a while.  I too get the 'error disconnected' when trying to send/save the location.  Can't do it when attached to the PC on web client (no option?).

**Reply by thatSFguy** [reply]
If you had a pre 2.5 release installed on the RAK, then do a full erase and fresh install of the firmware.  Reconfigure the node manually and *don't use a config backup*.  I've seen all kinds of issues with just an upgrade >2.5.

**Reply by thatSFguy** [reply]
P.S. Make sure you are on the latest phone app

**Comment by ohd22** [comment]
Hello and thanks for the info.  Your situation sounds really familiar.  My Units also generate a "phantom" location when trying to input my GPS coordinates (right aound 2.1 km to my south).  What trips me out is the fact this happens with all my devices (two HeltecV3's and one Wisblock).  I've yet to find thte common denominator!
    On Saturday, January 4, 2025 at 03:40:04 PM PST, LCuinu ***@***.***> wrote:  
 
 


Same problem on two RAK 4631. Entered a manual coordinate via bluetooth (using as routers in the hills). Initially the oordinates seem to take but after a while they change to a random location but within about 2.5km! When I ask for 'request position' (when linked to another device) the position is correct. But after a time it changes back to a location I've never ever visited or had aquaintance with (but quite near...about 2.5km). The first RAK I thought 'oh I'll just delete the DB on both' but it comes back!? All devices are on beta 2.5.15.7. Strangest thing. The second brand new unit setup tonight is in the middle of a Loch...until I request position but then goes back to the Loch adter a while. I too get the 'error disconnected' when trying to send/save the location. Can't do it when attached to the PC on web client (no option?).
